### ISSUE TITLE
ERROR window

### ISSUE THREAD

**Issue by jessetjazz** [Impro-Visor/Impro-Visor#61]
![Capture](https://user-images.githubusercontent.com/58867615/143070296-143c1e83-6a6e-4e12-a802-8c70dc388bcd.PNG)
 
Anyone have  a clue how to fix this issue please?  

Lost 3 years of work . I can NOT open my impro-visor files or the mainframe app

thank you

**Comment by lmorillas** [comment]
What has changed in your improvisor install? 

Work is not lost because, you have your improvisor edited files, don't you?

**Comment by jessetjazz** [comment]
Yes I am sure files of past work are still there it just not being able to get past / delete / remove the ERROR warning popup window , no matter how many times I uninstall and re- install IMPRO- VISOR set-up download
I open and copied to flash USB a NEW download on an old laptop and it open as a fresh / clean app, but when I transfer the Flash to my current workhorse laptop it will still install and open to those ""ERROR " pop ups 
 
![143070296-143c1e83-6a6e-4e12-a802-8c70dc388bcd](https://user-images.githubusercontent.com/58867615/143592573-f1a7b6b3-ae56-42fa-bbfa-be16080b49fb.PNG)

**Comment by xhevahir** [comment]
My guess would be that Impro-Visor is not getting permission to access files in that directory of your computer for some reason. Maybe try running the program as Administrator? Although I don't know how secure that is.

It looks like the configuration.install4j file does allow setting a custom directory for installation, but after that you'd have to go through the process of compiling the whole program.
